js/app/modules/card/list.js

```diff
@@ -100,6 +107,15 @@ const List = new Module.Class({
             height: image_height,
         });
         this._inner_content_grid.size_allocate(text_alloc);
+
+        let checkmark_alloc = new Gdk.Rectangle({
```

Comment:
You can probably skip this part if the checkmark isn't visible, for better performance.